src/app/app-extras.module.ts

```diff
@@ -21,3 +27,24 @@ import { StacheModule, StacheConfigService } from '@blackbaud/stache';
   entryComponents: []
 })
 export class AppExtrasModule { }
+
+(SkyAppBootstrapper as any).processBootstrapConfig = () => {
```

Comment:
Yes. This is a stopgap until the full implementation can get rolled out in SKYUX proper.